src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbTestRunner.java

```diff
@@ -485,93 +485,93 @@ void run() {
                     .withKeys(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue")))
                     .withOldImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue1")))
                     .withNewImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue1Updated")))),
-            new MTRecord()
+            new MtAmazonDynamoDb.MtRecord()
                 .withContext("ctx1")
-                .withTableName(getPrefix() + "MTAmazonDynamoDBTestRunner1")
+                .withTableName(getPrefix() + "MtAmazonDynamoDbTestRunner1")
                 .withEventName(MODIFY.name())
                 .withDynamodb(new StreamRecord()
                     .withKeys(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue")))
                     .withOldImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue1Updated")))
                     .withNewImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue1UpdatedAgain")))),
-            new MTRecord()
+            new MtRecord()
                 .withContext("ctx2")
-                .withTableName(getPrefix() + "MTAmazonDynamoDBTestRunner1")
+                .withTableName(getPrefix() + "MtAmazonDynamoDbTestRunner1")
                 .withEventName(MODIFY.name())
                 .withDynamodb(new StreamRecord()
                     .withKeys(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue")))
                     .withOldImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue2")))
                     .withNewImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValue2Updated")))),
-            new MTRecord()
+            new MtAmazonDynamoDb.MtRecord()
                 .withContext("ctx1")
-                .withTableName(getPrefix() + "MTAmazonDynamoDBTestRunner2")
+                .withTableName(getPrefix() + "MtAmazonDynamoDbTestRunner2")
                 .withEventName(INSERT.name())
                 .withDynamodb(new StreamRecord()
                     .withKeys(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue")))
                     .withNewImage(ImmutableMap.of("hashKeyField", new AttributeValue().withS("hashKeyValue"),
                         "someField", new AttributeValue().withS("someValueTable2")))),
-            new MTRecord()
+            new MtAmazonDynamoDb.MtRecord()
```

Comment:
why is this expanded to explicitly denote the outer class (i.e., "MtAmazonDynamoDb.MtRecord" instead of just "MtRecord")?